### DISCUSSION TITLE
RRSI

### DISCUSSION THREAD

**Discussion by Dragonlord174** [springbok/MLM2PRO-GSPro-Connector#150] [Q&A] [answered]
Good Day all,
I am using the MLM2PRO and trying to understand what is the best RRSI # as I get a -80 -- not sure is that good or bad.

Thanks
Dragonlord174

**Comment by wonder99** [comment, marked as answer]
That's quite bad. If you're getting disconnected, I recommend buying a
powered usb extender and plugging the Bluetooth adapter into it.  Place
that antenna within a foot of the MLM. You should be able to get something
in the -60 range.

References:
Alxum 32 Feet Active USB...
https://www.amazon.ca/dp/B094QNV1MJ?ref=ppx_pop_mob_ap_share

GAROGYI Super Long Range...
https://www.amazon.ca/dp/B0CB95RPFF?ref=ppx_pop_mob_ap_share

**Reply by Dragonlord174** [reply]
![thumbnail_IMG_2162](https://github.com/user-attachments/assets/58d4a4ce-2e1c-4e2a-96df-8f398f34b2ba)
Not sure if you can see it but got the RRSI down to -7 -- using ( buying) as per your reply

**Comment by Dragonlord174** [comment]
So a lower number is better— as I bought a Bluetooth adapter and I get around 25 RRSI

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Dragonlord174** [comment]
If I move my computer farther away I can get it down to-9

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by wonder99** [comment]
Great. That should color the rssi box green for you.
Yes, the less negative the better.

**Comment by Dragonlord174** [comment]
Thank you for  the advice and help— and for making the connector- much appreciated

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by wonder99** [comment]
Can you attach a screenshot of that? Did you get your MLM recently?

**Comment by Dragonlord174** [comment]
Yes— about 1 month ago— It been working great except for short chip shots- I wil take a screenshot of the  connection sometime soon as I won’t be able to  play for a day or two — again thank you

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by wonder99** [comment]
That's amazing, congrats!
